walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -223,6 +286,34 @@ public void justEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void justEnoughRoomWithoutOptionalAttendee() {
+    // Based on justEnoughRoom, add an optional attendee B who has an event between 8:30 and 8:45. 
+    // The optional attendee should be ignored since considering their schedule would result in a time slot smaller than the requested time.
+    //
+    // Events  : |--A--|     |----A----|
+    //              |--B--|
+    // Day     : |---------------------|
+    // Options :       |-----|
```

Comment:
nit: fix this ascii, this looks like A & B's first unavailable time overlaps, where B's start (8:30) should be aligned to the end of A's first slot.